gapic/samplegen/samplegen.py

```diff
@@ -13,29 +13,54 @@
 # limitations under the License.
 
 import itertools
+import jinja2
 import keyword
 import re
-import gapic.samplegen.utils
+
+from gapic.samplegen import utils
 
 from collections import (defaultdict, namedtuple)
-from typing import (List, Mapping, Set)
+from textwrap import dedent
+from typing import (List, Mapping, Set, Tuple)
 
 # Outstanding issues:
-# * Neither license nor copyright are defined
 # * In real sample configs, many variables are
 #   defined with an _implicit_ $resp variable.
 
 MIN_SCHEMA_VERSION = (1, 2, 0)
 
 VALID_CONFIG_TYPE = 'com.google.api.codegen.SampleConfigProto'
 
-# There are a couple of other names that should be reserved
-# for sample variable assignments, e.g. 'client', but there are diminishing returns:
-# the sample config author isn't a pathological adversary, they're a normal person
-# who may make mistakes occasionally.
-RESERVED_WORDS = frozenset(itertools.chain(keyword.kwlist, dir(__builtins__)))
-
-TEMPLATE_NAME = 'samplegen_template.j2'
+FILE_HEADER = {"copyrightLines": "Copyright (C) 2019  Google LLC",
```

Comment:
This seems awfully Google-specific. I avoided putting license stuff in these files (opting instead to have Yoshi add it) since the generator was supposed to be company-agnostic.

If we are to add these, why not put it in the template rather than here?

Reply:
The original intention was to read in copyright and license from input files, but somehow (can't remember where) I was convinced that would not be happening in the near future.
I'd rather just yank them and make it Yoshi's responsibility to add them then to add it to the template; keeping them as constants in python code makes it more straightforward to read them in from files later.

Reply:
After some second thoughts, I move to punt the issue and just remove the copyright and license until publishing samples and microgenerator created libraries is the topic at hand. We can then hash it out between us and Yoshi as to where the responsibility falls.

Reply:
I suggest that for the short-term we either keep the license as a placeholder or add a really obvious TODO. My main driver is ensuring that we do not accidentally publish Google libraries generated with this tool that lack the legal boilerplate.

Longer term, I think this makes it high priority to require that all microgenerators have a way of specifying a license file which will then be commented into every generated artifact. Counterargument: users can do this themselves, post-generation. Counter-counter argument: we want to make it easy for users to have generated artifacts, and since we're generating, it should be trivial for us to do it.

Reply:
+1 for making it standard that microgenerators add license and copyright information to samples.

Reply:
Well, right now, the microgenerator adds it to samples and not the main files. Let's definitely not do that. :-)

Can we back this portion out of this PR and make it a separate item, and determine what the right thing to do is?

Reply:
The license and copyright are now dummy values with TODO so that we don't forget.